VTEX - Orders API.json

```diff
@@ -19897,8 +19897,13 @@
                               },
                               "quantity": {
                                 "type": "integer",
-                                "description": "Quantity of the item being modified.",
+                                "description": "Quantity of the item being modified. Must be greater than 0.",
                                 "example": 1
+                              },
+                              "price": {
+                                "type": "integer",
+                                "description": "Price of the item being changed in cents, starting from 0.",
```

Comment:
I don't understand what "starting from 0" mean. Does it mean this value is necessarily equal or greater than 0? I recommend being clearer here.